cmd/kubeadm/app/cmd/init.go

```diff
@@ -122,6 +122,8 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 				kubeadmutil.CheckErr(err)
 			}
 
+			features.ExecFeatureGateChain(cfg.FeatureGates)
```

Comment:
Please call `ResolveFeatureGateDependencies` from `features.NewFeatureGate` so we don't have to duplicate these calls everywhere (there are many more usages of `features.NewFeatureGate` than this one